packages/scm/src/browser/scm-tree-widget.tsx

```diff
@@ -661,15 +665,14 @@ export class ScmResourceGroupElement extends ScmElement<ScmResourceGroupComponen
 export namespace ScmResourceGroupComponent {
     export interface Props extends ScmElement.Props {
         treeNode: ScmFileChangeGroupNode;
-        groupLabel: string;
```

Comment:
I believe we should document this as a breaking change.

Reply:
Happy to do so. As I mentioned in my description, I could also leave these fields untouched and just add `caption`, if that's preferable?